<launch>
  <node name="mcu_interface_node" pkg="shipbot_ros" type="mcu_interface_node.py" output="screen"/>
  <node name="chassis_control_node" pkg="shipbot_ros" type="dummy_chassis_node.py" output="screen"/>
</launch>
